CHANGELOG.md

```diff
@@ -37,7 +60,7 @@ These should not affect the functionality of the module.
 
 - resources fail to prefetch when rabbitmq is not intended to be installed \(via --noop or --tags\) [\#961](https://github.com/voxpupuli/puppet-rabbitmq/issues/961)
 
-## [v13.1.2](https://github.com/voxpupuli/puppet-rabbitmq/tree/v13.1.2) (2023-11-05)
+## [v13.1.2](https://github.com/voxpupuli/puppet-rabbitmq/tree/v13.1.2) (2023-11-06)
```

Comment:
I saw these date ones as well. Is this from forcing GMT as the timezone where we didn't before, or is it something where we can suppress it by setting a specific `TZ` explicitly when running?

Reply:
`git show v13.1.2` says:
```
tag v13.1.2
Tagger: William Yardley <wyardley@users.noreply.github.com>
Date:   Sun Nov 5 16:03:45 2023 -0800

Version 13.1.2
-----BEGIN PGP SIGNATURE-----

iQIzBAABCAAdFiEEU/h9uxV6/xQ8qSMcCHDrALalN+0FAmVILWEACgkQCHDrALal
N+1Ieg/+KCKnuF/mZsu6jgeGX7lwIvzLg/hnoUHgiQLWFzIUgGz45NIfXmUj4VQJ
mxsh+lhORxUdOzoA5FD/NIcP1WOAvqHjri1mvng1VmBCVS8FfGkNSSzaKLfNUa6x
BJbXx6sFGLZo4vLu5zuJSRcIfeu+p3wIaieneKBlIiE9ew0r7EOWUFQyjWD0qyum
/WQHIFhy8/cLtYBWrlXENUmknVMwY7kCNO15Rp2sNyhUpumR65CElIFHOmT8MU2G
TL4bVEOkHMPDQM8zmZfHpT5ZowRSlJKh8gyfb4D0roMen3vacvZu6HlIpEawJmAP
D4GXbIpMe+XzqABIMgwHTTuDWfahIIQlqQDsdQEA/S3poLR824Tmn7MiwVAZGE46
bfXntXIuBRUbC/G/dbGwXie28SVJDWyjoV3Mj1nRJ5ob73EK3/mCRt6OfEgVzrd/
rJe2X9Rl+XBcvSCdxWc6opVmg6Sm9D5XviHCcIyuNyx/pGHCaDUisM02pAXN1QPm
9TzVCQNSz3w9Qixwq744zp1TXOjBLr187D7qSevxVy6kOc5+c37Efb+T89FZkv4G
JNF8JpK3cAsjZO4YIB4/bb6TC/2tc1jGFKhd00hBXWF0bxanHY/STCW3d1q/yzuA
Udc5XhEZYRMuhPzLsWVg1Wx3iLrLotHp4Z39xRMUPFmJArJj12M=
=+RA7
-----END PGP SIGNATURE-----

commit e45e37fe345acb20b9a4afe499da8f22382fed40 (tag: v13.1.2)
Author: James Beattie <52940257+jimmybigcommerce@users.noreply.github.com>
Date:   Mon Nov 6 11:01:20 2023 +1100

    Release 13.1.2 (#959)
```

so while the tag is from 2023-11-05, the commit is from 2023-11-06. No idea how that's possible?

Reply:
ah no, both dates show different timezones? what? :D